### PR TITLE
move hard code to config file

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: world/world.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func (tw *TheWorld) GetGoldInfo() (common.GoldData, error) {
 }
 
 //NewTheWorld return new world instance
-func NewTheWorld(dpl deployment.Deployment, keyfile string) (*TheWorld, error) {
+func NewTheWorld(dpl deployment.Deployment, configFile string) (*TheWorld, error) {
```
Comment (secmask) [inline world/world.go]: we no longer need `SimulatedEndpoint`, as all url now load from config file, `dpl` param can remove also

File: cmd/configuration/config_files.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+)
+
+// NewSecretConfigCliFlag returns the cli flag to configure secret config file flag.
+func NewSecretConfigCliFlag() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:   secretConfigFileFlag,
+			Usage:  "location of secret config file",
+			EnvVar: "SECRET_CONFIG_FILE",
+			Value:  filepath.Join(common.CmdDirLocation(), "secret_config.json"),
```
Comment (secmask) [inline cmd/configuration/config_files.go]: we dont need to join `common.CmdDirLocation()`, as config file should be passed as service parameter.

File: cmd/configuration/config_files.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		},
+		cli.StringFlag{
```
Comment (secmask) [inline cmd/configuration/config_files.go]: we dont need to join common.CmdDirLocation(), as config file should be passed as service parameter.

Comment (NNagato) [reply cmd/configuration/config_files.go]: changed as request

File: cmd/configuration/config_files.go (hunk 2 above)
Comment (NNagato) [reply cmd/configuration/config_files.go]: changed as request

File: world/world.go (hunk 1 above)
Comment (NNagato) [reply world/world.go]: changed as request

File: world/endpoint.go [hunk 4]
```diff
@@ -87,7 +89,7 @@ type Endpoint interface {
 
 // RealEndpoint return real endpoint
 type RealEndpoint struct {
-	OneForgeKey string `json:"oneforge"`
+	EPS common.WorldEndpoints `json:"eps"`
```
Comment (halink0803) [inline world/endpoint.go]: Using "EPS" seems confusing, should use understandable name like ConfigEndpoints, AllEndpoints, etc.

Comment (NNagato) [reply world/endpoint.go]: PR updated in https://github.com/KyberNetwork/reserve-data/pull/811/commits/e2e8da039841591f996060b2fdfe23bdee90ff0f

**Comment by secmask** [comment]
@NNagato I update the issue with one more task, please check it out.

**Comment by NNagato** [comment]
> @NNagato I update the issue with one more task, please check it out.

PR updated, please help to review

**Pull request merged**